2023-03-07 22:25:43,230 DEBUG    root            Loaded Command Group: ['gcloud', 'config']
2023-03-07 22:25:43,231 DEBUG    root            Loaded Command Group: ['gcloud', 'config', 'config_helper']
2023-03-07 22:25:43,232 DEBUG    root            Running [gcloud.config.config-helper] with arguments: [--format: "json", --quiet: "True", --verbosity: "warning"]
2023-03-07 22:25:43,232 DEBUG    root            (gcloud.config.config-helper) You do not currently have an active account selected.
Please run:

  $ gcloud auth login

to obtain new credentials.

If you have already logged in with a different account:

    $ gcloud config set account ACCOUNT

to select an already authenticated account to use.
Traceback (most recent call last):
  File "/Users/auxia/Library/Application Support/cloud-code/installer/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 987, in Execute
    resources = calliope_command.Run(cli=self, args=args)
  File "/Users/auxia/Library/Application Support/cloud-code/installer/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 807, in Run
    resources = command_instance.Run(args)
  File "/Users/auxia/Library/Application Support/cloud-code/installer/google-cloud-sdk/lib/surface/config/config_helper.py", line 99, in Run
    cred = store.Load(use_google_auth=True)
  File "/Users/auxia/Library/Application Support/cloud-code/installer/google-cloud-sdk/lib/googlecloudsdk/core/credentials/store.py", line 565, in Load
    cred = _Load(
  File "/Users/auxia/Library/Application Support/cloud-code/installer/google-cloud-sdk/lib/googlecloudsdk/core/credentials/store.py", line 733, in _Load
    raise creds_exceptions.NoActiveAccountException(
googlecloudsdk.core.credentials.exceptions.NoActiveAccountException: You do not currently have an active account selected.
Please run:

  $ gcloud auth login

to obtain new credentials.

If you have already logged in with a different account:

    $ gcloud config set account ACCOUNT

to select an already authenticated account to use.
2023-03-07 22:25:43,233 ERROR    root            (gcloud.config.config-helper) You do not currently have an active account selected.
Please run:

  $ gcloud auth login

to obtain new credentials.

If you have already logged in with a different account:

    $ gcloud config set account ACCOUNT

to select an already authenticated account to use.
